File: app/src/main/java/com/neome/java/api/ent/entMain/sig/SigFormMappingResultGet.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entMain.sig;

import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdForm;
import com.neome.java.api.meta.base.dto.FormValue;
import com.neome.java.api.nucleus.base.sig.Sig;

import org.jetbrains.annotations.Nullable;

import java.util.Set;

@SuppressWarnings("unused")
public class SigFormMappingResultGet extends Sig
{
  public MetaIdForm formId;

  public FormValue formValue;

  @Nullable
  public Set<MetaIdField> outputFieldIdSet;
}

File: app/src/main/java/com/neome/api/ent/entMain/sig/SigFormMappingResultGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entMain.sig

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.dto.FormValue
import com.neome.api.nucleus.base.sig.Sig

open class SigFormMappingResultGet : Sig() {
    lateinit var formId: MetaIdForm
    lateinit var formValue: FormValue
    var outputFieldIdSet: Array<MetaIdField>? = null
}
